app/models/institution_type.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class InstitutionType < ApplicationRecord
+  # create a has many relationship with the institution model
```

Comment:
IMO this comment does not bring any value - `has_many :institutions` is self-documenting enough, no need to add any additional comments